src/app.py

```diff
@@ -55,10 +68,10 @@ def init():
         print("Check the log file for further information: " + LOG_FILE_NAME)
 
 
-
-@app.route('/', methods = ['GET'])
+@app.route('/', methods=['GET'])
 def index():
-    return "Hello! This is HuBMAP UUID API service :)"
+    return "Hello! This is the " + group_name + " UUID API service :)"
```

Comment:
Need None check here for `group_name` because you had the default value as `None`

Reply:
I now raise an exception in both app.py and uuid_worker when setting these values from the config file if "API_TYPE" is absent